cumulus/parachains/runtimes/assets/asset-hub-rococo/tests/tests.rs

```diff
@@ -1106,8 +1107,10 @@ mod asset_hub_rococo_tests {
 		let block_author_account = AccountId::from(BLOCK_AUTHOR_ACCOUNT);
 		let staking_pot = StakingPot::get();
 
-		let foreign_asset_id_location =
-			Location::new(2, [Junction::GlobalConsensus(NetworkId::Westend)]);
+		let foreign_asset_id_location = Location::new(
+			2,
+			[Junction::GlobalConsensus(NetworkId::ByGenesis(WESTEND_GENESIS_HASH))],
+		);
```

Comment:
Will also need a migration for the bridged assets registered as foreign assets on Rococo and Westend Asset Hubs.

Their IDs are changing now, storage items need migrating.

Reply:
> Will also need a migration for the bridged assets registered as foreign assets on Rococo and Westend Asset Hubs.
> 
> Their IDs are changing now, storage items need migrating.

Migrates storage to v5?

Reply:
cc @franciscoaguirre since he looked at same migration for v3->v4

Reply:
@franciscoaguirre pls provide more info or ref about migration

Reply:
We're going to merge this as is before the migration. We'll tackle the migration later. Created an issue to track it: https://github.com/paritytech/polkadot-sdk/issues/6319